### PR TITLE
feat: close sidebar after clicking on a menu item

### PR DESCRIPTION
**Pull request by idzeravianka**

### CODE REVIEW

**Comment by idzeravianka** [comment]
[Sidebar should close after navigation (mobile)](https://trello.com/c/1W1tGieI/36-sidebar-should-close-after-navigation-mobile)

File: js_machine_front/src/components/navBar/navBar.tsx [hunk 1]
```diff
@@ -1,41 +1,53 @@
...
-      <div className="logo-box">
-        <NavLink to="/">
-          <img src={LogoSrc} className="logo-box__link" alt="logo" />
-        </NavLink>
-      </div>
+export class NavBar extends React.PureComponent {
+  public state: NavBarState = {menuIsOpened: false};
+
+  handlingMenu = () => {
+    if (this.state.menuIsOpened) {
```
Comment (vharadkou) [inline js_machine_front/src/components/navBar/navBar.tsx]: consider to use this.setState(state => ({menuIsOpened: !state.menuIsOpened}));

File: js_machine_front/src/components/navBar/models/navBar.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export interface NavBarState {
+    menuIsOpened: boolean;
```
Comment (vharadkou) [inline js_machine_front/src/components/navBar/models/navBar.ts]: please rename to menuOpened

File: js_machine_front/src/components/navBar/navBar.tsx [hunk 3]
```diff
@@ -1,41 +1,53 @@
...
-    <div>
-      <div className="logo-box">
-        <NavLink to="/">
-          <img src={LogoSrc} className="logo-box__link" alt="logo" />
-        </NavLink>
-      </div>
+export class NavBar extends React.PureComponent {
+  public state: NavBarState = {menuIsOpened: false};
+
+  handlingMenu = () => {
```
Comment (vharadkou) [inline js_machine_front/src/components/navBar/navBar.tsx]: rename to handleMenu

**Review by vharadkou — CHANGES_REQUESTED** [review]

**Review by vharadkou — APPROVED** [review]

**Pull request merged**